client/my-sites/themes/controller.jsx

```diff
@@ -83,13 +83,16 @@ export function fetchThemeData( context, next ) {
 	}
 
 	const queryParams = {
-		search: context.query.s,
-		tier: context.params.tier,
-		filter: context.params.filter,
-		page: 0,
+		search: context.query.s || '',
+		tier: context.params.tier || '',
+		filter: context.params.filter || '',
 		perPage: PER_PAGE,
 	};
 
-	context.store.dispatch( query( queryParams ) );
-	context.store.dispatch( fetchNextPage( false ) ).then( () => next() );
+	context.store.dispatch(
+		query( {
```

Comment:
I'm not sure that making `query` also fetch the first page of content is the best solution — but one benefit is that it means setting up my state tree server-side now only requires a `query()` dispatch without also manually dispatching `fetchNextPage()`.